### ISSUE TITLE
cron to delete old records after X time passed

### ISSUE THREAD

**Issue by jpahullo** [ndunand/moodle-tool_mergeusers#144]
As an admin I want to set up a relative deadline after which any existing log older than this relative deadline is deleted.

### Notes

* It can be implemented as a programmed task to run daily or monthly.

### Acceptance conditions

* [ ] I see a new plugin's setting to define relative deadline, measured in number of days. 
* [ ] I see a programmed task that removes logs older (greater) than specified number of days.
* [ ] If I set up 0 days, logs are never removed.

**Comment by jpahullo** [comment]
I think, @danmarsden, this is something you told us to implement on the other issue, isn't it?

**Comment by danmarsden** [comment]
Yeah - makes sense to clean up logs after a time period - I need to implement this in some of my plugins too. :-)
